public/data/sumitmukharjeeeeee.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sumit Mukharjee",
+    "bio": "Learning and Exploring",
+    "avatar": "https://github.com/sumitmukharjeeeeee.png",
+    "links": [{
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/sumitmukharjeeeeee",
+        "icon": "github"
+    }, {
```

Comment:
Please move the next curly braces after the comma, to the next line.

Reply:
This is complete. ✅

Reply:
We can click the resolve button after the issues are fixed 🙂